python/activator/activator.py

```diff
@@ -588,8 +734,16 @@ def server_error(e: Exception) -> tuple[str, int]:
 def main():
     # This function is only called in test environments. Container
     # deployments call `create_app()()` through Gunicorn.
-    app = create_app()
-    app.run(host="127.0.0.1", port=8080, debug=True)
+    if platform == "knative":
+        _log.info("starting knative instance")
```

Comment:
This line is misleading, since it's not running a Knative instance. Maybe something like "Starting standalone Flask app?" Possibly the same comment about the other line; like I said, I'm confused about the workflow.

Reply:
It would be helpful if you could update the comment at the start of `main` to say how production deployments call `keda_start` (assuming they do).